components/HomeCare/DaySummary.js

```diff
@@ -153,7 +153,7 @@ const DaySummary = ({ daySummaryItem, edit = (item) => console.log(item), remove
                 <span>What should be done</span>
                 <p>
                   {whatShouldBeDone}{' '}
```

Comment:
remove {' '}

Reply:
replace similar code on {`${variable} `} where we need space